test/integration/bad/function-signatures/overloading/stanc.expected

```diff
@@ -0,0 +1,159 @@
+  $ ../../../../../../install/default/bin/stanc no_minimum_promotion.stan
+Semantic error in 'no_minimum_promotion.stan', line 12, column 9 to column 17:
+   -------------------------------------------------
+    10:  
+    11:  model {
+    12:     print(foo(1,2));
+                  ^
+    13:  }
+   -------------------------------------------------
+
+Ill-typed arguments supplied to function 'foo':
+(int, int)
+Available signatures:
+(real, int) => real
+  No unique minimum promotion found. Overloaded functions must not have
+  multiple equally valid promotion paths, but this function call has at least
+  two:
+  [int_to_real, None]
+  and
+  [None, int_to_real]
+  Consider defining a new signature for the exact types needed or re-thinking
+  existing definitions.
+(Additional signatures omitted)
```

Comment:
These errors just look weird. It should be something more like
```
Arguments with ambiguous types supplied to function 'foo':
(int, int)
No unique minimum promotion found. Overloaded functions must not have
multiple equally valid promotion paths but this function has signatures:
(real, int) => real
(int, real) => real
Consider defining a new signature for the exact types needed or re-thinking
existing definitions.
```

Reply:
Done, I went with a very similar style:

```
Semantic error in 'no_minimum_promotion3.stan', line 21, column 9 to column 20:
   -------------------------------------------------
    19:  
    20:  model {
    21:     print(foo3(1,2,3));
                  ^
    22:  }
   -------------------------------------------------

No unique minimum promotion found for function "foo3".
Overloaded functions must not have multiple equally valid promotion paths.
For args (int, int, int), this function has several:
(int, int, real) => real
(real, int, int) => real
(int, real, int) => real
Consider defining a new signature for the exact types needed or
re-thinking existing definitions.
```